### DISCUSSION TITLE
Outline path is really +5mm of real job size..?

### DISCUSSION THREAD

**Discussion by andymid3D** [terjeio/ioSender#393] [Q&A]
Hi, just a quick question, when using Outline, am I right in seeing that the outline coords are really 5mm further out than real job coords..? I ask as I had set up a job with 10mm safety margen in the original sketch but when doing the outline I saw the max coords where 5mm less than the safety margen that had been included in the sketch, and therefore 5mm more than the real job path..?
Just would be cool to get this confirmed, for a moment it was puzzling, as this job goes right to the limits of the material and clamps, so seeing it go INSIDE the safety margen (clamps on the outside), was worrying until realised was also OUTSIDE job paths..

Thanks!

And great software so far, still learning the ins-and-outs, but is capable of more than the client expected, and more than the original hardware/proprietary software were capable of!

Andy
:-)

**Comment by terjeio** [comment]
>  the outline coords are really 5mm further out than real job coords

Job coords from the CAD file or gcode? It could be the tool radius? Post the gcode if you want me to check this out.

**Reply by andymid3D** [reply]
am requesting the CAD file and gcode file from the client, but AFAIK, the actual job coords cutlines where about 5mm further in from the outline taken by the machine, whilst the CAD file had a 10mm margin over the cutlines... the cuts where ON the line, and with a 3.175mm bit, so tool radius would not seem to be the thing here... 
more than anything is curiosity.. ideally would get the client to make up an exact copy, but w/o the 10mm margin, and see what happens with both, but as have finished the install, don't think he is going to bother 🤷
